pkg/cstorvolume/v1alpha1/kubernetes.go

```diff
@@ -196,19 +255,34 @@ func (k *Kubeclient) GetRaw(name string, opts metav1.GetOptions) ([]byte, error)
 
 // List returns a list of cstor volume replica
 // instances present in kubernetes cluster
-func (k *Kubeclient) List(opts metav1.ListOptions) (*apis.CStorVolumeList, error) {
+func (k *Kubeclient) List(opts metav1.ListOptions,
+) (*apis.CStorVolumeList, error) {
 	cli, err := k.getClientOrCached()
 	if err != nil {
 		return nil, err
 	}
 	return k.list(cli, k.namespace, opts)
 }
 
-// Delete delete the cstorvolume resource
+// Delete deletes the cstorvolume resource
 func (k *Kubeclient) Delete(name string) error {
 	cli, err := k.getClientOrCached()
 	if err != nil {
 		return err
 	}
 	return k.del(cli, name, k.namespace, &metav1.DeleteOptions{})
 }
+
+// Create creates the cstorvolume resource
+func (k *Kubeclient) Create(vol *apis.CStorVolume,
+) (*apis.CStorVolume, error) {
```

Comment:
Does this require indentation? Is the signature going beyond 80 chars?
If yes, then can we have below indentation?
```go
func (k *Kubeclient) Create(
  vol *apis.CStorVolume,
) (*apis.CStorVolume, error)
```

Reply:
It did not require indentation